cmd/ocm-addons/installations/cmd.go

```diff
@@ -89,6 +93,13 @@ func run(opts *options) func(cmd *cobra.Command, args []string) error {
 				return err
 			}
 
+			if requiresSub {
```

Comment:
nit: why the extra indirection and not just? 
```go
if hasSubscriptionField(opts.Columns)
```

Reply:
Because the result doesn't change on subsequent executions. If I were to clean this up I'd probably pull the whole loop into it's own function (or a couple) and pass this as a parameter to keep the scope clean.

Reply:
Oh I did not see it was in a loop :sweat_smile: , gotcha !